chevah/github_hooks_server/handler.py

```diff
@@ -56,6 +47,33 @@ def ping(self, event):
             return 'Pong! But GitHub Zen text is missing.'
         return f'Pong! {zen}'
 
+    def pull_request(self, event):
+        """
+        Called, among other cases, when a PR review is requested.
+
+        Triggers the same actions as the needs-review command to issue_comment.
+
+        The `review_requested` action is under the `pull_request` event:
+        https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
```

Comment:
I am a bit confused. But this is due to GitHub API design and docs.
Your link is great.

I also found these 2 pages that are not that great
* https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#pullrequestevent
* https://docs.github.com/en/rest/reference/pulls


What is the difference between this method and `pull_request_review` ? 

I think that docstring should be brief. In the future it might not trigger the same actions as "needs-review".

For example, we might want to use the "draft" status to mark a PR as WIP - in work and then a draft action will trigger the same action as "needs-changes"

```suggestion
        Called for actions on a PR, exception the actions done by a reviewer.

        See: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
```

Reply:
Indeed, the difference is that `pull_request` is for **non-review** PR changes, and `pull_request_review` is for **reviews**. Your wording captures it.